distribution/docs/src/main/resources/_contents/_sources/catalog-providers-contents.adoc

```diff
@@ -1,8 +1,91 @@
+When the {branding} Solr is restarted, it reloads the data to get back the index.
+
+.Embedded Solr Provider Pros and Cons
+[cols="1,2a,2a,2a" options="header"]
+|===
+
+|Feature
+|Solr External Catalog Provider
+|Solr Embedded Catalog Provider
+|Solr Cloud Catalog Provider
+
+|Scalability
+|* Does not scale. Only runs one single server instance.
+|* Does not scale. Only runs one single server instance.
+|* Scalable
+ * Indexed documents can exceed ~2 billion
+
+|Flexibility
+|* The Solr Admin UI can be used.
+|* Requires no HTTP connection.
+ * The Solr Admin UI cannot be used.
+ * Allows for full control over the Solr Server.
+ No synchronous issues on startup; i.e., the Solr Server will synchronously start up with the Solr Provider
+ * Runs within the same JVM
+ * Setup and Installation is simple: unzip, modify system properties, and run.
+ * Can only be interfaced using Java
+|* The Solr Admin UI can be used.
+ * Requires HTTP connection.
+ * Requires Zookeeper to coordinate configurations between Solr Servers.
+ * Synchronous issues on startup; i.e., Zookeeper nodes needs to be up before Solr Cloud is up.
+ * Installation is more involved.
+
+|(Administrative)Tools
+|* Contains Solr Admin UI, which allows admins to query, check health, see metrics, see configuration files and preferences, etc.
+|* External open source tools like Luke will work to allow admins to check index contents
+  * JMX metrics reporting is enabled
+  * No Web Console. 
+  * No easy way to natively access (out of the box) what is in the index files or health of server at the data store level.
+|* Contains Solr Admin UI, which allows admins to query, check health, see metrics, see configuration files and preferences, etc.
+
+|Security
+|* Requires additional network protection to be secure.
+|* Does not open any ports which means no ports have to be secured.
+|* Requires additional network protection to be secure.
+ See Solr Cloud Hardening Guide for details.
+ 
+|Performance
+|* Requires HTTP or network overhead
+|* Requires no HTTP or network overhead
+|* Requires HTTP or network overhead
+ * Performance degrades as more data is indexed at ~2 billion documents
+
+|Backup/Recovery
+|* Backup command available.
+* Backups of indexes can be performed manually or through custom scripts
+|* Backups of indexes can be performed manually or through custom scripts
+  * Must copy files when server is shutdown
+|* Automatic recovery from failure.
+
+|===
+
+==== Solr Configuration Files
+
+The Apache Solr product has Configuration files to customize behavior for the Solr Server. These files can be found at `<DISTRIBUTION_INSTALLATION_DIRECTORY>/etc/solr`.
+Care must be taken in editing these files because they will directly affect functionality and performance of the Solr Provider.
+A restart of the distribution is necessary for changes to take effect.
```

Comment:
We're already warning them to do it carefully. Are you thinking we should explicitly tell them to not even try with Solr Cloud?